third_party/move/tools/move-linter/tests/model_ast_lints/nonminimal_bool.exp

```diff
@@ -0,0 +1,190 @@
+
+Diagnostics:
+warning: [lint] The left occurrence of `true` is redundant and should be removed.
+  ┌─ tests/model_ast_lints/nonminimal_bool.move:4:13
+  │
+4 │         if (true && x) ();
+  │             ^^^^^^^^^
+  │
+  = To suppress this warning, annotate the function/module with the attribute `#[lint::skip(nonminimal_bool)]`.
+  = For more information, see https://aptos.dev/en/build/smart-contracts/linter#nonminimal_bool.
```

Comment:
Also, right after this PR lands, the documentation should be updated so that this link here (or whatever it ultimately ends up being) works.